packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -70,12 +73,136 @@ export async function buildAction(projectDir: string, options: Options): Promise
   }
 }
 
+async function logBuildStatuses(
+  username: string,
+  buildStatus: {
+    jobs: Array<Record<string, any>>;
+    canPurchasePriorityBuilds: boolean;
+    numberOfRemainingPriorityBuilds: number;
+    hasUnlimitedPriorityBuilds?: boolean;
+  }
+) {
+  log('=================');
+  log(' Builds Statuses ');
+  log('=================\n');
+
+  buildStatus.jobs.forEach((job, i) => {
```

Comment:
We don't need to make the output of the new commands look exactly the same. IMO the output of the old command doesn't look great.
I think that we could use utils from `expo-cli/packages/expo-cli/src/commands/utils/cli-table.ts` to print a nice table with builds.
But feel free to propose sth.